flutter/news/test/core/network/network_info_test.mocks.dart

```diff
@@ -0,0 +1,82 @@
+// Mocks generated by Mockito 5.1.0 from annotations
```

Comment:
How does this mock generation work under the hood?

Can you point me to some documentation about it?

Reply:
I am asking because I wonder if you need to push this file, or if it's generated every time you run the test then we don't need pushing it to the repo. Anyway it will not get modified

Reply:
removed mocks from the github.
added command to generate mocks to github action
added .mocks.dart to gitignore 
https://github.com/novoda/multiplatform-playground/pull/9/commits/162e8639947659a91444dceaf0e700c936e00c31